includes/challenges-13.test.js

```diff
@@ -0,0 +1,273 @@
+'use strict';
+/* global describe, expect, test */
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 1
+//
+// Write a function named firstLetters that takes in an array of strings and returns an array
+// containing only the first letter of each string.
+//
+// For example, ['this is great :)', 'wow', 'whyyyyyy :(', ':)))))'] returns ['t', 'w', 'w', ':'].
+// ------------------------------------------------------------------------------------------------
+
+const firstLetters = (strs) => {
+  return strs.map(string => string.substring(0, 1));
+};
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 2
+//
+// Write a function named findHappiness that takes in an array of strings and returns an array
+// containing only the strings from the input array that contain ":)".
+// e.g. ['this is great :)', 'wow', 'whyyyyyy :(', ':)))))'] => ['this is great :)', ':)))))']
+// ------------------------------------------------------------------------------------------------
+
+const findHappiness = (strs) => {
+  return strs.filter(happy => happy.includes(':)'));
+};
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 3
+//
+// Write a function named standardizePhoneNumbers that takes in an array of phone number strings in
+// (XXX) XXX-XXXX format, and returns an array with the phone number strings in XXXXXXXXXX format.
+// ------------------------------------------------------------------------------------------------
+
+const standardizePhoneNumbers = (phoneNumbers) => {
+  return phoneNumbers.map(num => {
+    let newNum = num.substring(1, 4);
+    newNum += num.substring(6, 9);
+    newNum += num.substring(10, 14);
+    return newNum;
+  });
+};
+
```

Comment:
you can use template literal to do this in one line, `return phoneNumbers.map( num => `${num.substring(1,4)}${num.substring(6, 9)}${num.substring(10)}` );`